sdk/communication/Azure.Communication.CallingServer/src/autorest.md

```diff
@@ -13,7 +13,7 @@ If any of the new objects needs to be overwritten, add the required changes to t
 
 ```yaml
 require:
-    -  https://raw.githubusercontent.com/Azure/azure-rest-api-specs/5ffa6f90f1916423bd09855f1bfe32d2bd78ecab/specification/communication/data-plane/CallingServer/readme.md
+    -  https://github.com/Azure/azure-rest-api-specs/blob/b4b5fa5ee23f8cce9e1ade4a82076b4c34b25651/specification/communication/data-plane/CallingServer/readme.md
```

Comment:
please use raw file link
Steps:
Click Raw tab after you go into the perm link of that file.

Reply:
Fixed the path